README.md

```diff
@@ -99,6 +100,33 @@ For more details, please try [Getting started](./docs/en/getting-started.md).
   e.g. `cd khi`
 1. Run `cd ./web && npm install` from the project root
 
+#### Vale linting 
+We use Vale to enforce our documentation style and ensure consistency across our Markdown files.
+
+### Installation of Vale
+
+#### macOS
+    ```bash
+    brew install vale
+    ```
+
+#### Windows
+    ```bash
+    scoop install vale
+    ```
+
+#### Linux
+    ```bash
+    curl -L "https://github.com/errata-ai/vale/releases/download/v2.22.0/vale_2.22.0_Linux_64-bit.tar.gz" -o vale.tar.gz
+    tar -xzvf vale.tar.gz
+    sudo mv vale /usr/local/bin/vale
+    ```
+
+#### How to run Vale locally
+    ```bash
+    vale .
```

Comment:
Other development builds are written as Makefile build targets.
I would like to define a lint task for document as `lint-doc` as a new build target in Makefile.

I would have `lint-doc` defined in `./scripts/make/lint.mk` and it's should be dependant from `lint`  in `Makefile`.
References:
https://github.com/GoogleCloudPlatform/khi/blob/main/Makefile#L18
https://github.com/GoogleCloudPlatform/khi/blob/main/scripts/make/lint.mk#L1-L7